libraries/o-tracking/src/javascript/tracking.js

```diff
@@ -17,7 +17,7 @@ const initPage = page.init;
  *
  * @type {string}
  */
-const version = '4.5.0';
+const version = '4.5.3';
```

Comment:
o-tracking is currently 4.5.3 but, when this is released, it'll be 4.5.4 😅 

```suggestion
const version = '4.5.4';
```

Reply:
Oh, right! That's correct. But if we don't release it won't be published to npm, and won't be available to users.

Reply:
Oh, right! That's correct. However we'll need to release to get the version updated in production. Looks like it's send with tracking events and I guess that's what you want to update?

Reply:
Yes, that's correct. Thanks!

Reply:
Approved! 👍 I'll go ahead and release